static/prism.less

```diff
@@ -0,0 +1,41 @@
+div.code-toolbar {
+	line-height: 0;
+	> .toolbar {
+		line-height: 0;
+		top: 0 !important;
+		right: 0 !important;
+		border-left: 1px #dfdfdf solid;
+		border-bottom: 1px #dfdfdf solid;
+		> .toolbar-item {
+			line-height: 0;
+		}
+	}
+	@media (hover: none) {
+		> .toolbar {
+			opacity: 1 !important;
+		}
+	}
+
+}
+
+.toolbar a, 
+.toolbar button, 
+.toolbar span {
+	background: #eee !important;
+	background: rgba(238, 238, 238, 1) !important;
+	color: #858585 !important;
+	padding: .35em 2.25em !important;
+	box-shadow: none !important;
+	border-radius: 0 !important;
```

Comment:
Same thing for the `!important` in the rest of this file. We should avoid them if at all possible.